### PR TITLE
Adds docs on using enterprise plugin in CI/CD

### PR DESCRIPTION
**Pull request by skierkowski**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -212,4 +212,4 @@ $ sls deploy
 ```
 ## Next steps
 
-Learn more about the features and benefits of [Serverless Framework Enterprise](https://github.com/serverless/enterprise/blob/master/README.md)
+Learn more about the features and benefits of [Serverless Framework Enterprise](https://github.com/serverless/enterprise/blob/master/README.md) and how to [run Serverless Framework Enterprise in your CI/CD pipeline](./running-in-cdcd.md).
```
Comment (dschep) [inline docs/getting-started.md]: lol, why is the file name `...-cdcd.md` rather than `...-cicd.md`

File: docs/running-in-cdcd.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Serverless Framework Enterprise - Running in CI/CD
+
+For teams that use the Serverless Framework with the Enterprise Plugin to develop services it might be desirable to deploy the services from a CI/CD pipeline instead of the local CLI. Complete the steps in this guide to install the Serverless Framework open-source CLI, configure authentication for the Enterprise plugin, and deploy a service on AWS in your CI/CD service. This guide will walk you through each of the following:
+
+- [Install Node.js and NPM](#install-nodejs-and-npm)
```
Comment (dschep) [inline docs/running-in-cdcd.md]: These steps feel rather unnatural to me. I'm struggly to put into words exactly why, but it's mostly that some are steps you have to take in our system and the user's CD provider once(create access key, configure access keys for sls&aws). The other is part of the ci job's configuration (install nodejs/npm & sls).

File: docs/running-in-cdcd.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+5. You will be presented with the access key on the new page. 
+
+**Note**: The access token has permission to the tenant; however, it is associated with your account. If your account is deleted, then the access token will be revoked too.
+
+## Configure your Serverless Framework Enterprise access token in CI/CD
+
+In the previous step you obtained an access token from the Serverless Framework Enterprise dashboard which you will now set in your CI/CD environment such that the Serverless Framework open-source CLI can authenticate with the Serverless Framework Enterprise dashboard.
+
+```sh
+export SERVERLESS_ACCESS_KEY=<your-serverless-enterprise-access-key>
```
Comment (dschep) [inline docs/running-in-cdcd.md]: Setting env vars will never actually look like this in any CI system. Maybe screenshots of a common tool would be good? This is TravisCI for example:
image

File: docs/getting-started.md (hunk 1 above)
Comment (skierkowski) [reply docs/getting-started.md]: 🤦‍♂️

File: docs/running-in-cdcd.md (hunk 3 above)
Comment (skierkowski) [reply docs/running-in-cdcd.md]: I'll update this to be more generic. Each CI/CD service pretty much has it's own way of setting it (e.g. shell, config files, config UI).

File: docs/running-in-cdcd.md (hunk 2 above)
Comment (dschep) [reply docs/running-in-cdcd.md]: I like that 👍

**Pull request merged**